python/pyspark/ml/tests.py

```diff
@@ -594,17 +594,21 @@ def test_fit_minimize_metric(self):
         iee = InducedErrorEstimator()
         evaluator = RegressionEvaluator(metricName="rmse")
 
-        grid = (ParamGridBuilder()
-                .addGrid(iee.inducedError, [100.0, 0.0, 10000.0])
-                .build())
+        grid = ParamGridBuilder() \
+            .addGrid(iee.inducedError, [100.0, 0.0, 10000.0])\
```

Comment:
super minor nit here but I think generally we have ' \' rather than '\' for multi-line (as you've done elsewhere in this PR).